core/src/verify.rs

```diff
@@ -1508,6 +1557,61 @@ mod test {
         .unwrap_err();
     }
 
+    #[test]
+    fn test_verify_reserved_state_expelled_member() {
+        // Given: Configuration for the test
```

Comment:
I think it might be better to change the format of comments below, for example, just leave a sentence with additional explanation rather than use `Given: ` or `When: `.